From 45a302812dc691668eefe1daade9a6b05af99fed Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Mon, 30 Nov 2015 17:11:10 +0800
Subject: [PATCH 0716/5242] MLK-11921-3 mxc IPUv3: regs: Fix the macro WROD
 definition

commit  725e00a5398cd35b1112b0d332a4f113f6e4da0f from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch adds missing parentheses around the argument of the macro
WROD to avoid any potential macro expansion issue.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/ipu3/ipu_regs.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/ipu3/ipu_regs.h b/drivers/mxc/ipu3/ipu_regs.h
index 79051cf..099346d 100644
--- a/drivers/mxc/ipu3/ipu_regs.h
+++ b/drivers/mxc/ipu3/ipu_regs.h
@@ -696,7 +696,7 @@ enum di_sync_wave {
 };
 
 /* DC template opcodes */
-#define WROD(lf)		(0x18 | (lf << 1))
+#define WROD(lf)		(0x18 | ((lf) << 1))
 #define WRG	        	(0x01)
 
 #endif
-- 
1.7.9.5

